.gitignore

```diff
@@ -16,3 +16,5 @@
 .*.swp
 /tags
 /capistrano
+
+dump.rdb
```

Comment:
I think the dump.rdb is a Redis data storage thing that gets created on the fly... so I must have left this here from when we were using Sidekiq.